src/components/navigation-bar/navigation-bar.stories.mdx

```diff
@@ -120,4 +120,4 @@ top/bottom can be offset using the `offset` prop.
 
 ### Navigation Bar
 
```

Comment:
**question(non-blocking):** There appears to be a change here but the two lines look the same?